soc/arm64/renesas_rcar/gen3/pfc_r8a7795.c

```diff
@@ -536,3 +578,7 @@ const struct pfc_drive_reg *pfc_rcar_get_drive_regs(void)
 {
 	return pfc_drive_regs;
 }
+const struct pfc_pocctrl_reg *pfc_rcar_get_io_voltage_regs(void)
```

Comment:
Missing empty line before function declaration.

Did you ran `checkpatch`, BTW?

Reply:
checkpatch runs automatically here by GitHub
and this is the same code style as for other functions in this file

Reply:
done